JapaneseFilter/sections/specific.txt

```diff
@@ -15,6 +15,8 @@ kyoko-np.net##div[class^="col-"] > div.ad
 sankei.com###sponsored
 ganganonline.com##header#sqexHeader-black_rsp
 asahi.com##.AdMod
+japanese.engadget.com##p.partner_container
+japanese.engadget.com##aside#ldrBoardAd
```

Comment:
Please check if this rule required on engadget.com

Reply:
there are no such elements on engadget.com